src/test/java/org/zalando/planb/provider/AuthorizationCodeGrantFlowIT.java

```diff
@@ -54,7 +59,7 @@ public void invalidResponseType() {
                 .build();
 
         try {
-            ResponseEntity<String> response = getRestTemplate().exchange(request, String.class);
+            getRestTemplate().exchange(request, String.class);
             fail("GET should have thrown Bad Request");
         } catch (HttpClientErrorException ex) {
             assertThat(ex.getStatusCode()).isEqualTo(HttpStatus.BAD_REQUEST);
```

Comment:
as a minor detail, since you static imported HttpStatus.OK let's do maybe the same for BAD_REQUEST